### ISSUE TITLE
Black background isn't displayed correctly in PowerShell

### ISSUE THREAD

**Issue by alvinseville7cf** [microsoft/terminal#9115]
# Environment

```none
Windows build number: Win32NT 10.0.19042.0 Microsoft Windows NT 10.0.19042.0
Windows Terminal version (if applicable): 1.5.10271.0
```

# Steps to reproduce
If you execute this code peace than you see yellow text with white background instead of yellow text with black background.
```powershell
 Write-Host "TextToOutput" -ForegroundColor Yellow -BackgroundColor Black
```
![image](https://user-images.githubusercontent.com/42812113/107644949-d69d7d00-6cc3-11eb-9304-b0560b01f902.png)

# Expected behavior

Yellow text with black background.

# Actual behavior

Yellow text with white background.

**Comment by DHowett** [comment]
Unfortunately, this is by design. PowerShell erroneously tells us to use a black background _so often_ that it would cause there to be black streaks across the display (even if the terminal background was set to white or acrylic transparency.)

We had to disable black for PowerShell, specifically. #6810. Tracking the "fix" in /dup #6807.

**Comment by KalleOlaviNiemitalo** [comment]
I presume it is caused by <https://github.com/microsoft/terminal/pull/6810>, i.e. PowerShell asks for a black background when it really wants a default background, and Windows Terminal tries to work around that by recognising  the `powershell.exe` name and giving it the default background when it asks for a black background.
